.github/workflows/horusec-pr.yml

```diff
@@ -20,8 +20,6 @@ on:
     pull_request:
         branches:
             - main
-            - horusec
-            - release/*
```

Comment:
We should keep SAST validation on release branches according to our best practices validations.